pages/admin.tsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from "react";
+import { useRouter } from "next/router";
+
+import { Wrapper } from "../components/helper-components";
+import { useEditState } from "../utils/editState";
+
+const GoToEditPage: React.FC = () => {
+  const { setEdit } = useEditState();
+  const router = useRouter();
+  useEffect(() => {
+    setEdit(true);
+    router.back();
```

Comment:
What happens if I'm on the homepage and I hit edit with this pattern?

Reply:
Both the edit button and GoToEditPage do the same thing. They are just calling `setEdit(true)` So you would just go into edit mode as expected.